src/change/sdp.rs

```diff
@@ -743,7 +743,10 @@ fn ensure_stream_tx(session: &mut Session) {
 
         // If any payload param has RTX, we need to prepare for RTX. This is because we always
         // communicate a=ssrc lines, which need to be complete with main and RTX SSRC.
-        let has_rtx = session.codec_config.iter().any(|p| p.resend().is_some());
+        let has_rtx = session
+            .codec_config
+            .iter()
+            .any(|p| p.resend().is_some() && media.remote_pts().contains(&p.pt));
```

Comment:
Can you change this so that the `media.remote_pts()` check comes in a `.filter` before the `.any`?